tests/test-cross-context-dispatch/ccd-tests/src/test/resources/plans/context-ee10-forward-dump.json

```diff
@@ -0,0 +1,29 @@
+{
+  "class": "org.eclipse.jetty.tests.ccd.common.DispatchPlan",
+  "requestStep": {
+    "class": "org.eclipse.jetty.tests.ccd.common.steps.HttpRequestStep",
+    "method": "GET",
+    "requestPath": "/ccd-ee10/redispatch/ee10"
+  },
+  "steps": [
+    {
+      "class": "org.eclipse.jetty.tests.ccd.common.steps.ContextRedispatchStep",
+      "dispatchType": "FORWARD",
+      "contextPath": "/ccd-ee10",
+      "dispatchPath": "/dump/ee10"
+    }
+  ],
+  "expectedEvents": [
+    "Initial plan: context-ee10-forward-dump.json",
+    "DispatchPlanHandler.handle() method=GET path-query=/ccd-ee10/redispatch/ee10",
+    "org.eclipse.jetty.tests.ccd.ee10.CCDServlet.service() dispatcherType=REQUEST method=GET requestUri=/ccd-ee10/redispatch/ee10",
+    "org.eclipse.jetty.tests.ccd.ee10.DumpServlet.service() dispatcherType=FORWARD method=GET requestUri=/ccd-ee10/dump/ee10"
+  ],
+  "expectedProperties": [
+    {
+      "class": "org.eclipse.jetty.tests.ccd.common.Property",
```

Comment:
We also need to be able to test object wrapper identity (see  #11888). So we need a test that wraps the request before the dispatch, and inside the dispatch target we see exactly the same object.

Reply:
That's done in a different testcase right now.

See `RedispatchTests.testEe8FilterWithAwkwardRequestURI()`

Reply:
I don't think that test is actually testing object wrapper identity with an `==` but rather inferring it: I think Greg has said he'd like to see an actual instance equality test.